Cue/app/tabs/library/LibraryHome.js

```diff
@@ -142,18 +142,25 @@ class LibraryHome extends React.Component {
   _onPressLookUpDeckByShareCode = () => {
     CuePrompt.prompt(
       Platform.OS === 'android' ? 'Look up share code' : 'Look Up Share Code',
-      '',
+      'Spaces and capitalization aren’t important.',
```

Comment:
I don't think it's 100% necessary, but it's extra assurance for users. Kind of like the reverse of how so many password fields still tell you passwords are "case-sensitive".